workshop_3/workshop_3.ino

```diff
@@ -0,0 +1,56 @@
+#include <Adafruit_Sensor.h> // include these three libraries (Adafruit_Sensor.h, DHT.h, DHT_U.h)
+#include <DHT.h> 
+#include <DHT_U.h>
+DHT_Unified dht(2, DHT11);   // to define that sensor DHT11 is on pin 2
+const int RedLedPin = 13; // to define that the red LED is on pin 13
```

Comment:
Variable name not ok : `redLedPin` but if you put a define it will be `#define RED_LED_PIN`